server/store/sqlstore/migrations.go

```diff
@@ -37,7 +37,7 @@ const (
 
 func (s *SQLStore) runMSTeamUserIDDedup() error {
 	// get all users with duplicate msteamsuserid
-	rows, err := s.getQueryBuilder().Select(
+	rows, err := s.getMasterQueryBuilder().Select(
```

Comment:
Why is master needed here for a SELECT?

Reply:
@wiggin77 I assumed is needed as this is a migration to avoid any sort of lag. I can change it to use the replica if you think is best

Reply:
I'll change it to use the replica and then merge.. if this causes issues we can always switch to use master again